src/Sql/Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -30,6 +30,7 @@ namespace Microsoft.Azure.Commands.Sql.Database.Cmdlet
     /// <summary>
     /// Cmdlet to create a new Azure Sql Database
     /// </summary>
+    [GenericBreakingChange("Database property 'BackupStorageRedundancy' has been changed to 'CurrentBackupStorageRedundancy' and 'RequestedBackupStorageRedundancy' in this version.")]
```

Comment:
Rephrase:
Database property 'BackupStorageRedundancy' has been deprecated in this version. Use 'CurrentBackupStorageRedundancy' and 'RequestedBackupStorageRedundancy' instead.

Reply:
The cmd line input argument is still BackupStorageRedundancy; the return values are now Current/RequestedBackupStorageRedundancy instead of BackupStorageRedundancy

Reply:
Maybe I misunderstood this. Why do you need to add the breaking change phrase here then?

Reply:
This is required for breaking changes.  I can clarify the return value part